packager/packager.cc

```diff
@@ -545,7 +545,7 @@ ast::ParsedFile rewritePackage(core::Context ctx, ast::ParsedFile file, const Pa
     }
 
     // Sanity check: __package.rb files _must_ be typed: strict
-    if (file.file.data(ctx).strictLevel < core::StrictLevel::Strict) {
+    if (file.file.data(ctx).originalSigil < core::StrictLevel::Strict) {
```

Comment:
We set the `strictLevel` lower artificially for autogen so we don't get errors even from invalid code. That means that in autogen every `__package.rb` file would raise this error. We can sidestep that by checking the `originalSigil`, which is retained even if `strictLevel` is set lower.